nixos/modules/programs/gpu-screen-recorder.nix

```diff
@@ -34,6 +34,8 @@ in {
       capabilities = "cap_sys_nice+ep";
       source = "${package}/bin/gpu-screen-recorder";
     };
+
+    environment.etc."modprobe.d/gsr-nvidia.conf".source = "${package}/lib/modprobe.d/gsr-nvidia.conf";
```

Comment:
I just need a confirmation here, it seems like nixpkgs makes no attempt to automatically link any files from a packages `lib/modprobe.d` and `etc/modprobe.d` to the systems `/run/modprobe.d` or `/etc/modprobe.d`, correct?